Scripts/Core/VRView.cs

```diff
@@ -151,7 +151,8 @@ public void OnEnable()
 			// VRSettings.enabled latches the reference pose for the current camera
 			var currentCamera = Camera.current;
 			Camera.SetupCurrent(m_Camera);
-			VRSettings.enabled = true;
+			UnityEngine.XR.XRSettings.enabled = true;
+			UnityEngine.XR.InputTracking.Recenter();
```

Comment:
This crept back in. Do we need it for 2017.2 now?

For clarity, this was removed to fix the "workspaces are at a weird rotation" bug, #197

Reply:
No, it can go